src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Everything/IPatientEverythingService.cs

```diff
@@ -0,0 +1,26 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Health.Fhir.Core.Features.Search;
+using Microsoft.Health.Fhir.Core.Models;
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations.Everything
+{
+    public interface IPatientEverythingService
+    {
+        Task<SearchResult> SearchAsync(
+            string resourceType,
```

Comment:
>string resourceType, [](http://example.com/codeflow?start=12&length=20)

I don't think you need it. This is I`Patient`EverythingService. You can use const string inside instead of passing parameter.

Reply:
Agree. It is removed in the update.